flask_restx_patched/parameters.py

```diff
@@ -111,6 +111,9 @@ def validate_patch_structure(self, data):
         is prepended with '/'.
         Removing '/' in the beginning to simplify usage in resource.
         """
+        if 'op' not in data:
```

Comment:
This is required as if the op is not in the module OPERATION_CHOICES, this lookup causes a KeyError that is not caught (from what I could see) anywhere in the call stack. 
Not certain that this is the best fix (the calling code knows there's an error but still continues) but it is a fix